angular/playground/projects/ng-forms-manager/src/lib/forms-manager.ts

```diff
@@ -17,6 +17,18 @@ export interface AkitaAbstractGroup<C = any> extends AkitaAbstractControl {
 
 export type ArrayControlFactory = (value: any) => AbstractControl;
 
+export type FormsManagerOptions = {
+  debounceTime: number;
+  updateStoreOnUnsubscribe: boolean;
+};
+
+export const FORMSMANAGER_OPTIONS = new InjectionToken<FormsManagerOptions>('FormsManagerOptions');
+
+const defaultOptions: Partial<FormsManagerOptions> & { debounceTime: number; updateStoreOnUnsubscribe: boolean } = {
```

Comment:
Why we need the `& { debounceTime: number; updateStoreOnUnsubscribe: boolean }`?

Reply:
I followed the structure that was used in the akita devtools for config injection.